apps/meteor/app/channel-settings/server/functions/saveRoomType.js

```diff
@@ -41,13 +41,13 @@ export const saveRoomType = function (rid, roomType, user, sendMessage = true) {
 	if (sendMessage) {
 		let message;
 		if (roomType === 'c') {
-			message = TAPi18n.__('Channel', {
+			message = TAPi18n.__('Public', {
 				lng: (user && user.language) || settings.get('Language') || 'en',
-			});
+			}).toLowerCase();
```

Comment:
not sure, but maybe it is a better idea to have different translations for lowercase.. not sure how other languages behave to `toLowerCase()`

Reply:
updated!